src/plugins/discover/public/application/application.ts

```diff
@@ -7,25 +7,35 @@
  */
 
 import './index.scss';
-import angular from 'angular';
+import { renderApp as renderReactApp } from './index';
 
 /**
  * Here's where Discover's inner angular is mounted and rendered
  */
 export async function renderApp(moduleName: string, element: HTMLElement) {
-  await import('./angular');
-  const $injector = mountDiscoverApp(moduleName, element);
-  return () => $injector.get('$rootScope').$destroy();
+  const app = mountDiscoverApp(moduleName, element);
+  return () => {
+    app();
+  };
 }
 
-function mountDiscoverApp(moduleName: string, element: HTMLElement) {
-  const mountpoint = document.createElement('div');
+function buildDiscoverElement(mountpoint: HTMLElement) {
+  // due to legacy angular tags, we need some manual DOM intervention here
```

Comment:
do we still need those?

Reply:
yes. that or we need to update a bunch of functional tests that are checking for presence of `discover-app` tag

Reply:
fine with that, can be cleanup up later, when removing remaining stuff